packaging/setup_osx.py.in

```diff
@@ -0,0 +1,73 @@
+import sys
+import os.path
+
+from cx_Freeze import setup, Executable, hooks
+
+def load_scipy(finder, module):
+    finder.IncludePackage("scipy._lib")
+    finder.IncludePackage("scipy.misc")
+    finder.IncludePackage("scipy.sparse.csgraph._validation")
+
+hooks.load_scipy = load_scipy
+
+def load_pyqt5_qtquick(finder, module):
+    finder.IncludeModule("PyQt5.QtCore")
+    finder.IncludeModule("PyQt5.QtGui")
+    finder.IncludeModule("PyQt5.QtQml")
+    finder.IncludeModule("PyQt5.QtNetwork")
+    finder.IncludeModule("PyQt5._QOpenGLFunctions_2_0")
+    
+hooks.load_PyQt5_QtQuick = load_pyqt5_qtquick
+
+search_path = sys.path.copy()
+search_path.insert(1, "@EXTERNALPROJECT_INSTALL_PREFIX@/lib/python3.5/site-packages/")
+search_path.insert(2, "@CMAKE_INSTALL_PREFIX@/lib/python3.5/site-packages/")
```

Comment:
The python documentation for sys.path states:

> As initialized upon program startup, the first item of this list, path[0], is the directory containing the script that was used to invoke the Python interpreter. If the script directory is not available (e.g. if the interpreter is invoked interactively or if the script is read from standard input), path[0] is the empty string, which directs Python to search modules in the current directory first. Notice that the script directory is inserted before the entries inserted as a result of PYTHONPATH.

I did not want to change that assumption which is why I insert things after that entry.